src/data.js

```diff
@@ -0,0 +1,114 @@
+import * as Render from './renders';
+
+function parseWeatherData(data) {
+  return {
+    city: data.name,
+    temp: data.main.temp,
+    weather: data.weather[0].main,
+    weather_desc: data.weather[0].description,
+    min_temp: data.main.temp_min,
+    max_temp: data.main.temp_max,
+    humidity: data.main.humidity,
+    lat: data.coord.lat,
+    lon: data.coord.lon,
+  };
+}
```

Comment:
[REQUIRED] Please try to use ES6 syntax, I'm marking this lines of code, but all the functions in this file could apply the same modifications.
Remember object destructuring, and arrow functions from the previous lesson
![image](https://user-images.githubusercontent.com/73254040/96968097-19013b00-14d6-11eb-9a06-9592f8e78ef0.png)
[source](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)